### ISSUE TITLE
The steps of "Running locally" is outdated

### ISSUE THREAD

**Issue by mitsuruog** [styleguidist/site#10]
I tried to run this site locally by this step, but I could not get to run.

- https://github.com/styleguidist/site#running-locally

Some scripts in `react-styleguidist` are outdated.
Please let me know the correct step then I'll update README.
Thanks.

**Comment by sapegin** [comment]
I've just tried and it worked. What's not working for you?

**Comment by mitsuruog** [comment]
Let me try again! thank you for your quick response.

**Comment by mitsuruog** [comment]
Ah... maybe I was missing something. I tried it once again then it totally works.
Sorry to annoy you with such a weird topic.
